tools/ci_build/build.py

```diff
@@ -317,6 +319,14 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
                  "-Donnxruntime_USE_MKLDNN=" + ("ON" if args.use_mkldnn else "OFF"),
                  "-Donnxruntime_USE_MKLML=" + ("ON" if args.use_mklml else "OFF"),
                  "-Donnxruntime_USE_NGRAPH=" + ("ON" if args.use_ngraph else "OFF"),
+                 "-Donnxruntime_USE_OPENVINO=" + ("ON" if args.use_openvino else "OFF"),
+                 "-Donnxruntime_USE_OPENVINO_BINARY=" + ("ON" if args.use_openvino == "MYRIAD_FP16" else "ON"),
```

Comment:
should it be else "OFF" for onnxruntime_USE_OPENVINO_BINARY

Reply:
We have disabled building from source temporarily because of an issue with plugin library paths. Once we fix this, this will be changed.

Reply:
Cleaned this code to always use only the binary option for OpenVINO EP and never to use the source option.